index.md

```diff
@@ -238,3 +238,67 @@ Our typical usages of AWS include:
 1. S3 for hosting uploaded files
 1. ElastiCache instances for hosting in-memory data stores
 1. OpsWorks for provisioning each if the size or complexity is large
+
+# Documentation
+
+Traditionally documentation has been a second-class citizen in most Rails project. This holds true just as much outside The Nerdery as internally. As innovators and leaders in our industry we should hold ourselves to a higher standard and therefore provide ample documentation on the products we produce. A great article on improving Rails based documentation published by Aaron Sumner can be found [here](http://everydayrails.com/2016/04/18/rails-documentation-practices.html) and many of the practices described below are pulled from his philosophy.
```

Comment:
Do we want to include a section on using Swagger to document APIs? I think in the future we will see more Rails API projects and touching on API documentation could be a good idea.